### PR TITLE
Sync metadata job

### LINKED ISSUE #651: Sync metadata job for OpenManage

### PR DESCRIPTION
**Pull request by Pengpengwanga**
Close #651 

Signed-off-by: Pengpeng Wang <pengpengw@vmware.com>

### CODE REVIEW

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java [hunk 1]
```diff
@@ -135,18 +197,70 @@ private void syncMetricsDataJob(FacilitySoftwareConfig integration) {
 
    }
 
+   public List<Asset> handleServerAssets(DevicesResult<Server> serversResult, Map<Long,Asset> assetNumberMap ){
+      List<Server> servers = serversResult.getValue();
+      List<Asset> serverAssetsToSave = new ArrayList<Asset>();
+      for(Server server:servers) {
+         //filter by device status
+         if(DeviceStatus.NORMAL.getValue() != server.getStatus()) {
```
Comment (yixingjia) [inline openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java:205]: status. we should save the data as far as the server still there.

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java [hunk 2]
```diff
@@ -184,7 +297,59 @@ private void checkAndUpdateIntegrationStatus(FacilitySoftwareConfig integration,
       updateIntegrationStatus(integration);
    }
 
-   public OpenManageAPIClient createClient(FacilitySoftwareConfig integration) {
-      return new OpenManageAPIClient(integration);
+   private boolean valueIsChanged(String oldValue, String newValue) {
+      String value = oldValue == null ? "" : oldValue;
+      String comparedValue = newValue == null ? "" : newValue;
+      if(value.equals(comparedValue)) {
```
Comment (yixingjia) [inline openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java:303]: return !value.equals()

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java (hunk 1 above)
Comment (Pengpengwanga) [reply openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java]: done

File: openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java (hunk 2 above)
Comment (Pengpengwanga) [reply openmanage/src/main/java/com/vmware/flowgate/openmanage/job/OpenManageJobService.java]: Done

**Pull request closed without merging**